src/main/java/space/obminyashka/items_exchange/controller/UserController.java

```diff
@@ -105,6 +105,21 @@ public Page<AdvertisementTitleDto> getFavoriteAdvertisements(
         return advService.findAllFavorite(authentication.getName(), PageRequest.of(page, size));
     }
 
+    @PreAuthorize("hasAnyRole('USER', 'ADMIN', 'MODERATOR')")
+    @DeleteMapping(value = ApiKey.USER_DELETE_MY_FAVORITE, produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Delete user's favorite advertisement")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "202", description = "ACCEPTED"),
```

Comment:
Accepted is mostly used for PUT requests, and I recommend you to use 204 NO_CONTENT for your particular case.